src/storage/txn/commands/flashback_to_version.rs

```diff
@@ -75,8 +75,8 @@ impl<S: Snapshot, L: LockManager> WriteCommand<S, L> for FlashbackToVersion {
             &mut reader,
             &mut next_lock_key,
             &mut next_write_key,
-            self.key_locks,
-            self.key_old_writes,
+            self.key_locks.clone(),
```

Comment:
why clone

Reply:
Otherwise, the closure below will use a partially moved value: `self`.